activerecord/lib/active_record/model_schema.rb

```diff
@@ -595,6 +603,18 @@ def load_schema!
           super
         end
 
+        def _run_load_schema_callbacks # :nodoc:
+          @_load_schema_callback_already_called ||= false
```

Comment:
I suspect we actually don't want these checks, but I nonetheless put them here for discussion, assuming the interface `_run_load_schema_callbacks` is desirable